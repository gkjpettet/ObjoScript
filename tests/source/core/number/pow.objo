System.print(2.pow(2)) // Expect 4
System.print(0.pow(2)) // Expect 0
System.print(3.pow(2.5)) // Expect 15.588457268
System.print(2.pow(-3)) // Expect 0.125