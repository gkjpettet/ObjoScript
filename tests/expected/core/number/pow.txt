4
0
15.588457268
0.125